com.unity.multiplayer.mlapi/Editor/PostProcessScene.cs

```diff
@@ -11,13 +11,7 @@ public class NetworkScenePostProcess : MonoBehaviour
         [PostProcessScene(int.MaxValue)]
         public static void ProcessScene()
         {
-            //If we are in playmode (editor or stand alone) we do not want this to execute
-            if (Application.isPlaying)
-            {
-                return;
-            }
-
-            var traverseSortedObjects = FindObjectsOfType<NetworkObject>().ToList();
+            List<NetworkObject> traverseSortedObjects = MonoBehaviour.FindObjectsOfType<NetworkObject>().ToList();
```

Comment:
Is this still change on purpose? I'd say just `FindObjectsOfType` is more clean?

Reply:
yeah, I think this change should be reverted and stay as is with the original here. (probably merge artifacts)